assets/marketplace/cloudformation/Makefile

```diff
@@ -0,0 +1,217 @@
+
+
+# VPC ID used for builds
+BUILD_VPC_ID ?=
+
+# VPC subnet used for builds
+BUILD_SUBNET_ID ?=
+
+# Override for AMI name if needed
+BUILD_AMI_NAME ?=
+
+# Default build region
+AWS_REGION ?= us-west-2
+
+# Teleport version
+TELEPORT_VERSION ?= 3.0.1
+
+# Teleport UID is the UID of a non-privileged 'teleport' user
+TELEPORT_UID ?= 1007
+
+# Instance type is a single value, sorry
+INSTANCE_TYPE ?= t2.micro
+
+# Use comma-separated values without spaces for multiple regions
+# For now, limit AMI to regions with DynamoDB encryption at rest enabled
+# https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/EncryptionAtRest.html
+DESTINATION_REGIONS ?= us-west-2,us-east-1,us-east-2,eu-west-1
+
+# Cloudformation stack name to create, e.g. test1
+STACK ?=
+
+# Stack parameters, e.g ParameterKey=KeyName,ParameterValue=KeyName ParameterKey=DomainName,ParameterValue=teleport.example.com ParameterKey=DomainAdminEmail,ParameterValue=admin@example.com ParameterKey=HostedZoneID,ParameterValue=AWSZONEID
+STACK_PARAMS ?=
+
+# License S3 URI for enterprise builds, e.g. s3://example.com/path/to/license
+TELEPORT_LICENSE_URI ?=
+
+# Generate timestamp for builds
+BUILD_TIMESTAMP := $(shell TZ=UTC /bin/date "+%Y%m%d-%H%M%S%Z")
+
+# Jenkins test CloudFormation stack name
+JENKINS_TEST_STACK_NAME ?= jenkins-teleport-quickstart-test-stack
+
+# Jenkins Cloudformation parameters path
+JENKINS_CLOUDFORMATION_PARAMETERS_PATH ?= ./files/build/jenkins-parameters.json
+
+# YAML filename
+CF_YAML_FILENAME ?= ./oss.yaml
+
+# S3 Bucket ID
+S3_BUCKET_ID ?=
+export
+
+.PHONY: oss
+oss: TELEPORT_TYPE=oss
+oss:
+	@echo "Building image $(TELEPORT_VERSION) $(TELEPORT_TYPE)"
+	@echo "BUILD_TIMESTAMP=$(BUILD_TIMESTAMP)"
+	packer build -force -var build_timestamp=$(BUILD_TIMESTAMP) template.json
+	@echo "$(BUILD_TIMESTAMP)" > files/build/oss_build_timestamp.txt
+
+.PHONY: oss-name-build
+oss-name-build: TELEPORT_TYPE=oss
+oss-name-build:
+	@echo "Building image $(TELEPORT_VERSION) $(TELEPORT_TYPE) with name $(BUILD_AMI_NAME)"
+	packer build -force -var ami_name=$(BUILD_AMI_NAME) template.json
+
+.PHONY: ent
+ent: TELEPORT_TYPE=ent
+ent: check-ent-vars
```

Comment:
Can you leave a comment here what is this target about? I am not sure I followed.